t/version-detection.t

```diff
@@ -0,0 +1,56 @@
+use Mojo::Base -strict;
+use Test::Mojo;
+use Test::More;
+use JSON::Validator;
+
+use Mojolicious::Lite;
+get '/person'           => 'person';
+
+my $t = Test::Mojo->new;
+$t->get_ok('/person.json')->status_is(200);
```

Comment:
You don't really have to test this. It's done implicit by `load_and_validate_schema()`.

Reply:
Again all of this was copied directly from what you did in t/id-keyword-draft7.t. I am not a big fan of the style; it is very different from what I would have done in my own code. I copied it thinking it was the style you preferred.